geoportal/demo_geoportal/locale/de/LC_MESSAGES/demo_geoportal-client.po

```diff
@@ -357,7 +359,7 @@ msgstr "Information"
 
 #, fuzzy
 msgid "internal_id"
-msgstr "Extern"
+msgstr ""
```

Comment:
?

Reply:
I removed the translation, because it made no sense.

Reply:
Extern not but internal_id shouldn't be translated?... for me yes

Reply:
OK